security-analysis/security-analysis-api/src/main/java/com/powsybl/security/LimitViolationDetector.java

```diff
@@ -36,6 +36,19 @@ default void checkCurrent(Contingency contingency, Branch branch, Branch.Side si
         checkCurrent(branch, side, currentValue, consumer);
     }
 
+    /**
+     * Mirror checkCurrent on {@link Branch} but for {@link ThreeWindingsTransformer} instead.
+     *
+     * @param contingency   The contingency for which current must be checked, {@code null} for N situation.
+     * @param transformer           The three windings transformer on which the current must be checked.
+     * @param side          The side of the three windings transformer on which the current must be checked.
+     * @param currentValue  The current value to be checked, in A.
+     * @param consumer      Will be fed with possibly created limit violations.
+     */
+    default void checkCurrent(Contingency contingency, ThreeWindingsTransformer transformer, ThreeWindingsTransformer.Side side, double currentValue, Consumer<LimitViolation> consumer) {
+        checkCurrent(transformer, side, currentValue, consumer);
+    }
+
```

Comment:
Add the code but for now I am not able to do the test for checkAll because of lack of knowledges of how to create a good network with 3 windings transformers. It seems current network have NaN as p and q for transformer terminals. (see commit "Add three windings transformers check in checkAll for AbstractContingencyBlindDetector and AbstractLimitViolationDetector")

Reply:
I have done a test with some values maybe not realistic but it allows code coverage